src/gui/folderstatusmodel.cpp

```diff
@@ -157,9 +159,14 @@ QVariant FolderStatusModel::data(const QModelIndex &index, int role) const
         const auto supportsSelectiveSync = x._folder && x._folder->supportsSelectiveSync();
 
         switch (role) {
-        case Qt::DisplayRole:
+        case Qt::DisplayRole: {
             //: Example text: "File.txt (23KB)"
-            return x._size < 0 ? x._name : tr("%1 (%2)").arg(x._name, Utility::octetsToString(x._size));
+            const auto &xParent = static_cast<SubFolderInfo *>(index.internalPointer());
+            const auto suffix = (x._isNonDecryptable && x._checked && (!xParent || !xParent->_isEncrypted))
+                ? tr(" - %1").arg("Could not decrypt!")
+                : QString{};
+            return x._size < 0 ? QString(x._name + suffix) : QString(tr("%1 (%2)").arg(x._name, Utility::octetsToString(x._size)) + suffix);
```

Comment:
I know it's not something you changed here, but single letter variables are an abomination